dev/benchmarks/complex_layout/test_driver/semantics_perf_test.dart

```diff
@@ -31,7 +46,13 @@ void main() {
       await driver.forceGC();
 
       final Timeline timeline = await driver.traceAction(() async {
-        expect(await driver.setSemantics(true), isTrue);
+        expect(
+          await driver.setSemantics(true),
```

Comment:
is it possible to make it return true if semantics is already on?

Reply:
We could, but that defeats the purpose of the test. We want to make sure that this call actually turned on semantics so we can measure the cost of the initial semantics tree generation.

Reply:
ah you are right. I missed the point

Reply:
but if we do that we can just use `await driver.setSemantics(false)` instead of the one off adb command? One thing I worry about is this API seems like a foot gun that its return value is not really useful unless you force killed the a11y.

Reply:
`await driver.setSemantics(false)` is also not guaranteed to turn a11y off because there may be other entities (like accessibility services running on the platform) that force semantics to on. That's why `setSemantics` has the return value to tell you whether the call changed anything. It's not the greatest API design, I agree with that. If you have a suggestion of how to make this better, let me know (and file an issue).

Reply:
that explains a lot thanks